src/components/PrintModal/WatermarkModal/WatermarkModal.js

```diff
@@ -337,14 +364,37 @@ class WatermarkModal extends React.PureComponent {
               <div className="form-field">
 
                 <label>
-                  {t(`option.watermark.colors`)}
+                  {t(`option.watermark.style`)}
                 </label>
-                <div
-                  data-element="currentColorCell"
-                  className="cell"
-                  style={{ backgroundColor: formInfo[FORM_FIELD_KEYS.color].toHexString() }}
-                  onClick={() => this.setColorPaletteVisibility(!this.state.isColorPaletteVisible)}
-                >
+                <div className="style-container">
+                  <div
+                    data-element="currentColorCell"
+                    className="Button"
```

Comment:
Hmm do you think it make sense that we change the Button component so that it's possible to just render a div with nothing inside? If we do that you can just use a `<Button />` here

Reply:
I believe we are already doing that .... ?
![image](https://user-images.githubusercontent.com/54084743/70280510-76c94f00-176d-11ea-8fe2-31da7a144730.png)

For my case, content is undefined, so it ends up being a div with nothing inside...?

As for a button component that just contains a div ... ? I don't think I have seen that before and it sounds a little bit weird?

I think if the concern here is this not being a button component when it should be, I think we can just add an extra prop called styles which is of type CSS style?

Reply:
Hmmm sorry, didn't know that we are already doing this... I think it's ok to just use a Button component here. I'm not very sure what you mean by `I think we can just add an extra prop called styles which is of type CSS style`. Perhaps you can elaborate a bit more?

Reply:
Here is what I mean,
1. Create a new prop in the Button component called styles;
2. use that prop inside the button component

Example usuage:
``` 
<button
 styles={ {color: 'blue'} }
/>

button.jsx
<Button style={styles}></Button>
```

Reply:
Sorry I missed your reply.

I see what you mean now. But if you chose to use a `Button` component here, you don't need to use the `style` prop because the `Button` component has the `Button` class and you are styling it in the scss file(https://github.com/PDFTron/webviewer-ui/pull/432/files#diff-43b4e2cfdcecbbc6852c6706e5302a27R113).

I think we generally want to avoid using inline css because a customer would need to use `!important` to override it if he's using the custom `css` option. Anyways I think this PR is good to be merged in without this change. Currently just waiting for https://github.com/XodoDocs/webviewer/pull/495 to be merged.